pkgs/tools/security/gnome-keysign/default.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, fetchFromGitLab, python3, wrapGAppsHook
+, gobject-introspection, gpgme
+, gtk3, glib, gnome3, gst_all_1 }:
+
+let
+  # TODO: try to use multiple outputs
+  python-gpg = gpgme.override {
+    pythonSupport = true;
+    python = python3;
+  };
+in python3.pkgs.buildPythonApplication rec {
+  pname = "gnome-keysign";
+  version = "1.0.1";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "GNOME";
+    repo = pname;
+    rev = version;
+    sha256 = "0iy70dskd7wly37lpb2ypd9phhyml5j3c7rzajii4f2s7zgb3abg";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+  buildInputs = [
+    gobject-introspection # TODO: move to nativeBuildInputs after https://github.com/NixOS/nixpkgs/issues/56943 is fixed
+    # TODO: add avahi support
+    gtk3 glib
+    gst_all_1.gstreamer gst_all_1.gst-plugins-base gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-bad # TODO: requires zbar https://github.com/NixOS/nixpkgs/pull/54398
+  ];
+  propagatedBuildInputs = with python3.pkgs; [
+    lxml
```

Comment:
I see `lxml` and `Babel` as `setup_requires`.
So wouldn't these be `nativeBuildInputs`?